test/simple.spec.coffee

```diff
@@ -270,6 +270,18 @@ define ['chai', 'cs!./test-config'], ({assert, expect}, {client, USERNAME, TOKEN
         itIsOk(REPO, 'git.trees', 'c18ba7dc333132c035a980153eb520db6e813d57', 'fetch')
         # itIsOk(REPO, 'git.trees.create', {tree: [sha], base_tree: sha})
 
+        it '.git.refs.tags.fetch()', () ->
+          STATE[GH].repos('philschatz', 'octokat.js').git.refs.tags.fetch().then (tags) ->
```

Comment:
Thanks for adding the tests! The test repo is hidden to anyone but the octokit test user (which GitHub has marked as a spambot) but this is fine